core/templates/dev/head/player/PlayerServices.js

```diff
@@ -692,3 +692,63 @@ oppia.directive('feedbackPopup', ['oppiaPlayerService', function(oppiaPlayerServ
     }]
   };
 }]);
+
+oppia.controller('InformationCardCtrl', function ($scope, $modal) {
+
+  $scope.showDialog = function () {
+
+    var modalInstance = $modal.open({
+      animation:true,
```

Comment:
single space after : 

(Perhaps we should find a way to try and fix these automatically.)